mappings/net/minecraft/block/ChiseledBookshelfBlock.mapping

```diff
@@ -2,4 +2,15 @@ CLASS net/minecraft/class_7714 net/minecraft/block/ChiseledBookshelfBlock
 	FIELD field_40305 BOOKS_STORED Lnet/minecraft/class_2758;
 	FIELD field_40306 LAST_INTERACTION_BOOK_SLOT Lnet/minecraft/class_2758;
 	METHOD method_45457 tryRemoveBook (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_1657;Lnet/minecraft/class_7716;)Lnet/minecraft/class_1269;
+		ARG 0 world
+		ARG 1 pos
+		ARG 2 player
+		ARG 3 blockEnitity
```

Comment:
Enitity? Same below

Reply:
Didn't know convention. I'll fix it once I get home.

Reply:
Just to note, Technici4n pointed out a spelling error.

Reply:
Yes, `blockEntity` is fine (but not `blockEnitity`)

Reply:
Ah. I see. Kinda unclear